client/payment-details/dispute-details/dispute-notice.tsx

```diff
@@ -0,0 +1,71 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import NoticeOutlineIcon from 'gridicons/dist/notice-outline';
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import BannerNotice from 'components/banner-notice';
+import { reasons } from 'wcpay/disputes/strings';
+import { Dispute } from 'wcpay/types/disputes';
+import { isInquiry } from 'wcpay/disputes/utils';
+
+interface DisputeNoticeProps {
+	dispute: Dispute;
+	urgent: boolean;
+}
+
+const DisputeNotice: React.FC< DisputeNoticeProps > = ( {
+	dispute,
+	urgent,
+} ) => {
+	const clientClaim =
+		reasons[ dispute.reason ]?.claim ??
+		__(
+			'The cardholder claims this is an unrecognized charge.',
+			'woocommerce-payments'
+		);
+
+	const noticeText = isInquiry( dispute )
+		? /* translators: %s is the clients claim for the dispute, eg "The cardholder claims this is an unrecognized charge." */
```

Comment:
Do we need to mention what <a> is in the translators comment?

Reply:
Not sure what you mean here. Are you asking why I went into detail?

I included a sample of the inserted text so translators have the full context of what is being inserted as this may alter the placement. 

In other places, we have just inserted a dispute reason which can be enclosed in quotes. As this is a complete sentence we are inserting, I am assuming context is important. 

That said, I'm no language or i18n expert! So feel free to propose changes.

Reply:
🤪 There was another part of that comment that was stripped out somehow (using VSCode to PR review), apologies!

I meant to ask:

Do we need to mention what **`<a>`** is in the translators comment?

Reply:
Here's an example of the `<a>` being mentioned in the translators comment:

https://github.com/Automattic/woocommerce-payments/blob/610e3ad911307d45ba83529101d2337b77898464/client/account-status-settings/index.js#L53